### PR TITLE
asitop 0.0.24 (new formula)

### PR DESCRIPTION
**Pull request by b-reich**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This PR is still in Progress. Not sure how to test properly. Maybe someone has an Idea.

### CODE REVIEW

File: Formula/asitop.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    url "https://files.pythonhosted.org/packages/5e/5f/1e4bd82a9cc1f17b2c2361a2d876d4c38973a997003ba5eb400e8a932b6c/wcwidth-0.2.6.tar.gz"
+    sha256 "a5220780a404dbe3353789870978e472cfe477761f06ee55077256e509b156d0"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    pipe_output("#{bin}/asitop", "q", 0)
```
Comment (chenrui333) [inline Formula/asitop.rb:42]: anything to assert?

Comment (b-reich) [reply Formula/asitop.rb:42]: Its hard...
It looks like htop. Just the help menu.

Comment (SMillerDev) [reply Formula/asitop.rb:42]: You can start it and stop it again.

**Comment by chenrui333** [comment]
#154266

**Pull request closed without merging**